pkg/geo/parse.go

```diff
@@ -113,6 +125,9 @@ func parseGeoJSON(
 	if err := geojson.Unmarshal(b, &t); err != nil {
 		return geopb.SpatialObject{}, err
 	}
+	if soType == geopb.SpatialObjectType_GeographyType {
+		makeValidGeographyGeom(t)
```

Comment:
move all this into `spatialObjectFromGeomT`, just before we calculate the bounding box. this ensures we only do it in one place.

Reply:
Shouldn't we move it before we `ewkb.Marshal(geom.T)` ? Because I would imagine change in geom.T to change ewkb.